src/components/SimpleModal/SimpleModal.jsx

```diff
@@ -156,6 +159,7 @@ function SimpleModal({
     >
       <SimpleModalUI
         aria-modal="true"
+        aria-hidden={!isNil(rest.ariaHidden) ? rest.ariaHidden : !show}
```

Comment:
You can use the [nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) instead of bringing in a lodash function. I believe the same outcome can be accomplished with
```
aria-hidden={rest.ariaHidden ?? !show}
```

Reply:
I'm aware and sometimes used to use something different to do the same (`something != null`). This is very explicit and what we use throughout the library. Maybe we can eventually replace all uses of lodash.isNil with that, thanks for the suggestion!